src/main/java/xyz/niflheim/stockfish/engine/util/Util.java

```diff
@@ -0,0 +1,47 @@
+package xyz.niflheim.stockfish.engine.util;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.io.IOException;
+import java.nio.file.DirectoryStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Comparator;
+import java.util.Map;
+import java.util.TreeMap;
+
+public class Util {
+    private static final Log log = LogFactory.getLog(Util.class);
+
+    public static final String ASSETS_LOCATION = "assets/engines/";
+    public static final String ENGINE_FILE_NAME_PREFIX = "stockfish_";
+    public static final String ENGINE_FILE_NAME_SUFFIX = "_X64";
+    public static final String FILE_MASK = ENGINE_FILE_NAME_PREFIX + "??" + ENGINE_FILE_NAME_SUFFIX + "*";
+
+    public static Map<String, Integer> SUPPORTED_VERSIONS = new TreeMap<>(Comparator.reverseOrder());
```

Comment:
Why is it necessary to use treeMap here? I haven't found any use of the keys from this map anywhere. If this is necessary, then it would be better to use a TreeSet, or if you still need the keys, then swap the signature (Map<Integer, String)

Reply:
Good point.
Initially, I wanted to use such a map to validate file paths generated in getPath() against it, but then I decided it's not really needed. Converting to a set, as per your request.